Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/AppUpdaterCommandFactory.cs

```diff
@@ -119,13 +120,30 @@ public ICheckDiskSpace CreateCheckDiskSpaceCommandForDiff(int versionId, AppUpda
             }
 
             AppDiffSummary diffSummary = context.App.RemoteMetaData.GetDiffSummary(versionId, cancellationToken);
-            return new CheckDiskSpaceCommand(diffSummary, context.App.LocalDirectory.Path, biggestFileSize);
+            
+            var appDownloadDirectory = context.App.DownloadDirectory;
```

Comment:
do not use `var` is resulting type is unclear

Reply:
done